rigex-front-end/src/components/auth/signUpForm.js

```diff
@@ -25,9 +25,12 @@ const SignUpForm = () => {
     email: '',
     password: '',
     showPassword: false,
+    showHelpText: false,
   });
   const [errorMessage, setErrorMessage] = useState('');
-  const [helpMessage, setHelpMessage] = useState('');
```

Comment:
What was the need for this change?

Reply:
The field's onFocus wouldn't revert when the field was no longer active. I had to add another prop to the field for onBlur and this state needed to toggle depending on if the field was focused or not

Reply:
Shouldn't this message only show up when there's a problem with the user input?

Reply:
the designs show it appears when the field is focused. Barrette included the comment it should disappear when it is not in focus.